decidim-initiatives/config/locales/en.yml

```
@@ -146,8 +146,12 @@ en:
             time_and_date: Time and date
             timestamp: Timestamp
       titles:
+        edit_initiative_type_scope: Edit initiative type scope
         initiatives: Initiatives
+        initiatives_settings: Initiative settings
         initiatives_types: Initiative types
+        new_initiative_type: New initiative type
+        new_initiative_type_scope: New initiative type scope
```

Consistency with the view title: 
```suggestion
        new_initiative_type_scope: Create initiative type scope
```